webpack.config.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path')
+
+module.exports = {
+  entry: './lib/browser.js',
```

Comment:
nice, I figured the webpack config could be relatively small.

Reply:
They have rather nice docs, I found.